File: modul 3/Bahasa C/PRAK304-2210817110014-Muhammad_Firas.c
#include<stdio.h>

int main() 
{
    int bil;
    scanf("%d", &bil);

    switch (bil) 
    {
        case 1 ... 9:
            printf("Satuan");
        break;
        case 0:
            printf("Nol");
        break;
        case 100 ... 1000:
            printf("Anda Menginput Melebihi Limit Bilangan");
        break;
        case 20 ... 99:
            printf("Puluhan");
        break;
        case 10 ... 19:
            printf("Belasan");
        break;
    }
}

File: modul 3/Bahasa Python/PRAK304-2210817110014-Muhammad Firas.py
angka = int(input(""))

if (angka>0) and (angka<10):
    print("Satuan")

elif (angka>99):
    print("Anda Menginput Melebihi Limit Bilangan")

elif (angka>=20<100):
    print("Puluhan")

elif (angka>=10<20):
    print("Belasan")

else:
    print("Nol")
